examples/GLUE/src/infer.py

```diff
@@ -3,7 +3,7 @@
 
 def load_classifier():
     config = CustomArgParser().parse()
-    checkpoint_path = '/home/azureuser/GLUE/checkpoints/model_9.pt'
+    checkpoint_path = 'checkpoints/model_9.pt'
```

Comment:
@krishansubudhi Modified this to relative directory. I git cloned pymarlin and tried the example from within examples/glue. Let me know if you think its due to the I tried the example.

Reply:
This definitely due to my approach of cloning entire pymarlin. Still feel relative directory would work for all VM.

Reply:
necessary. Thanks